plugins/entando-plugin-jpkiebpm/src/main/resources/spring/plugins/jpkiebpm/aps/jpkiebpmServicesConfig.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file needed for? Are the services not picked up as part of the E5 package scans?

Reply:
some managers have already been injected but others are new.